project-addons/spain_account_invoice/models/account_invoice.py

```diff
@@ -21,6 +21,10 @@ def _report_xls_fields(self, cr, uid, invoice_type, context=None):
                 'date_invoice', 'number', 'supplier_number', 'partner_vat', 'partner_name', 'tax_base',
                 'tax_description', 'tax_amount', 'tax_amount_ret', 'amount_total', 'country_name'
             ]
+        elif invoice_type == 'intrastat':
```

Comment:
Esto no se como lo queréis hacer, peor tened en cuenta que los países tienen un check "Miembro Intrastat" quizás podáis filtrar por el país de Cliente/Proveedor de la factura